sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_builtin_evaluators.py

```diff
@@ -761,16 +735,16 @@ def test_multimodal_evaluator_content_safety_json_b64_image(self, project_scope,
         score = evaluator(conversation=conversation)
 
         assert score is not None
-        assert score["violence"] == "Very low"
+        assert score["violence"] is not None
```

Comment:
Maybe it's better to check if score["violence"] is a valid member of the HarmSeverityLevel enum.

```suggestion
        assert score["violence"] in (level.value for level in HarmSeverityLevel)
```